### PR TITLE
Integrate Coloured Styling for Introduction Section

### PR DESCRIPTION
**Pull request by samuelsogbesan**
- Add colouring to text and background
- Add padding to paragraph
- Shorten "introduction" navigation data-link attribute to match section id ("Intro")

### CODE REVIEW

File: portfolio/src/main/webapp/style.css [hunk 1]
```diff
@@ -145,6 +151,7 @@ body {
 .intro-wrapper p {
   background-color: white;
   text-align: center;
+  padding:1rem;
```
Comment (Somae) [inline portfolio/src/main/webapp/style.css]:
```suggestion
  padding: 1rem;
```

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -93,11 +93,16 @@ body {
...
+#intro {
+  background: var(--primary);
+}
+
 #intro,
 .intro-wrapper {
   align-items: center;
   display: flex;
   justify-content: center;
+  color:white;
```
Comment (Somae) [inline portfolio/src/main/webapp/style.css]:
```suggestion
  color: white;
```

Comment (Somae) [inline portfolio/src/main/webapp/style.css]: Nit: You might want to sort this in alphabetically as well.

File: portfolio/src/main/webapp/style.css [hunk 3]
```diff
@@ -124,15 +129,16 @@ body {
 }
 
 .intro-wrapper h3 {
-  font: lighter italic "Roboto Light";
-  margin:0;
+  font: italic 2rem Roboto Light;
```
Comment (Somae) [inline portfolio/src/main/webapp/style.css]: The style guide suggests using single quotes to enclose font names with spaces.

File: portfolio/src/main/webapp/style.css [hunk 4]
```diff
@@ -124,15 +129,16 @@ body {
...
+  margin: 0;
 }
 
 .intro-wrapper > * :first-child {
   flex: 0.2;
 }
 
 .intro-wrapper > * :last-child {
+  color:var(--primary);
```
Comment (Somae) [inline portfolio/src/main/webapp/style.css]:
```suggestion
  color: var(--primary);
```

**Review by Somae — CHANGES_REQUESTED** [review]

File: portfolio/src/main/webapp/style.css (hunk 3 above)
Comment (samuelsogbesan) [reply portfolio/src/main/webapp/style.css]: Thanks, rectified this in commit 7e0f486

File: portfolio/src/main/webapp/style.css (hunk 2 above)
Comment (samuelsogbesan) [reply portfolio/src/main/webapp/style.css]: Thanks, alphabetised the attributes with commit 48d2ccf

**Review by Somae — APPROVED** [review]

**Pull request merged**